### ISSUE TITLE
wkhtmltopdf inside a docker container : "Error: write EPIPE"

### ISSUE THREAD

**Issue by uraquitanfilho** [wkhtmltopdf/wkhtmltopdf#5062]
**wkhtmltopdf version(s) affected**: 0.4.0

**OS information**
Docker alpine image

**Description**
I have an application nodejs running using docker. If try to generate PDF by rest API (calling this container (have installed wkhmlttopf) take this error message: "Error: write EPIPE"

**Comment by PhilterPaper** [comment]
I'm not familiar with using Docker, but I _have_ heard of wkHTMLtoPDF getting confused about where resources are when piping in or out. If wkHTMLtoPDF is buried several levels down under Docker and Nodejs, this could be a problem to untangle. Can you run wkHTMLtoPDF from the command line? If so, will it run under Nodejs? Maybe you can isolate the problem that way.

**Comment by uraquitanfilho** [comment]
yes, all works inline inside the container. I did another container only to wkhtmltopdf as well but ... the best way is to works inside the API Rest container
